bin/xcop

```diff
@@ -21,6 +21,31 @@ def config(path)
   args
 end
 
+def expand_directories_to_xml_files(paths)
+  expanded = []
+  paths.each do |path|
+    if File.directory?(path)
+      expanded.concat(Dir.glob(File.join(path, '**', '*.xml'), File::FNM_DOTMATCH))
```

Comment:
@literally-bug-creator why only `*.xml`? We have `opts[:include]` that can be used here same way you use `opts[:exclude]` in the `apply_exclude_patterns()`. The user might call:

```
xcop . --include=foo/*.xsl --exclude=foo/bar/
```

This means that they want all `*.xsl` files in the `foo/` directory, but don't want any files from the `foo/bar/`.